python/python/glide/constants.py

```diff
@@ -26,7 +26,8 @@
 TRequest = Union[RedisRequest, ConnectionRequest]
 # When routing to a single node, response will be T
 # Otherwise, response will be : {Address : response , ... } with type of Dict[str, T].
-TClusterResponse = Union[T, Dict[str, T]]
+TClusterResponse = Union[T, Dict[bytes, T]]
+TClusterDecodedResponse = Union[T, Dict[str, T]]
```

Comment:
This is going to change/break the API everywhere...

Reply:
The purpose of this PR is to make this change

Reply:
+1

Reply:
In the next PR, the API signatures will be changed